src/components/details/ProductDetail.jsx

```diff
@@ -0,0 +1,60 @@
+
+import ProductView from './ProductView'
+import Products from '../Products/Products';
+
+import './ProductDetail.scss'
+
+import { useParams } from 'react-router-dom'
```

Comment:
Fix the import sequence

Reply:
fixed the import sequence according to your previous comment